mozilla/foundation.mozilla.org
# Updated donate links with UTM params

Closes #7099 

**Steps to test**:

1. Visit the [review app](https://foundation-s-7099-donat-ofucu6.herokuapp.com/en/)
2. click the "Donate" Link in the top right corner of the page
3. verify that the URL you have been redirected to has UTM parameters attached to it.
4. Visit the [review app](https://foundation-s-7099-donat-ofucu6.herokuapp.com/en/) again
5. Scroll down to reveal the footer, and click the "donate" link.
6. Verify that the URL you have been redirected to has UTM parameters appended to it.
7. If they work, testing is complete! 

## Checklist

**Changes in Models:**
- [x] [Are my changes backward-compatible](https://github.com/mozilla/foundation.mozilla.org/blob/master/docs/workflow.md#django-migrations-what-to-do-when-working-on-backward-incompatible-migrations). If not, did I schedule a deploy with the rest of the team?
